honorary-one-line-wonders/cpp/helloWorld.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+int main() 
+{
+    std::cout << "Hello, World!";
```

Comment:
See above comment, same thing applies. We can merge them into the honorary folder if you want to though.

Reply:
I thought it should be a bit more complicated than this so I put it in honorary.
But if you think it should be in the one-liners, I'm fine with that too :)

Reply:
We don't have a complexity requirement to be more beginner friendly 👍 (#19)